TrainingProject/src/main/java/com/example/trainningspringproject/service/MachineService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.trainningspringproject.service;
+
+import com.example.trainningspringproject.entity.Machine;
+import com.example.trainningspringproject.repository.MachineRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MachineService {
+    @Autowired
+    private MachineRepository Repo;
```

Comment:
the variable name should be useful, as when you read it inside the code you can understand it